### ISSUE TITLE
ToolTip toggles on Ctrl+Shift+Alt+F10

### ISSUE THREAD

**Issue by SamBent** [dotnet/wpf#4618]
* .NET Core Version: 3.0+
* Windows version: all
* Does the bug reproduce also in WPF for .NET Framework 4.8?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? no
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/master/README.md#reporting-security-issues-and-security-bugs).

 
 
 **Problem description:**
1. Run sample app from #4612
3. Press TAB repeatedly to move focus to the second Image
4. Press Ctrl+Alt+Shift+F10
 
 **Actual behavior:** 
The Image's ToolTip toggles on or off (perhaps after a delay due to #4615)

 **Expected behavior:**
 Nothing

Ctrl+Shift+F10 toggles the tooltip.  If extra modifier keys (e.g. Alt) are pressed, no toggle should occur.

**Comment by SamBent** [comment]
Fixed by the tooltip work in 6.0-preview7.
